Formula/f3.rb

```diff
@@ -16,10 +16,13 @@ class F3 < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:    "55da39f3758797df44426b7744542b4322ddc84b20fc7a5664e2da672cef0d1a"
   end
 
-  depends_on "argp-standalone"
+  on_macos do
+    depends_on "argp-standalone"
+  end
 
   def install
-    system "make", "all", "ARGP=#{Formula["argp-standalone"].opt_prefix}"
+    args = OS.mac? ? "ARGP=#{Formula["argp-standalone"].opt_prefix}" : nil
+    system "make", "all", args
```

Comment:
Looks like this is expanding to empty string input. Maybe just use an array and `*args`?